Python/sysmodule.c

```diff
@@ -211,7 +211,7 @@ sys_audit_tstate(PyThreadState *ts, const char *event,
 
     /* Initialize event args now */
     if (argFormat && argFormat[0]) {
-        eventArgs = _Py_VaBuildValue_SizeT(argFormat, vargs);
+        eventArgs = Py_VaBuildValue(argFormat, vargs);
```

Comment:
`sysmodule.c` doesn't define PY_SSIZE_T_CLEAN, but `Py_VaBuildValue` is now alias to `_Py_VaBuildValue_SizeT`.